.github/workflows/ci.yml

```diff
@@ -0,0 +1,81 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - prod
+      - dev
+  pull_request:
+    branches:
+      - master
+      - prod
+
+env:
+  GO111MODULE: on
+  TEST_TAGS: integration
+  DATABASE_MIGRATIONS_URL: file://$(pwd)/migrations
+  GEMINI_SUBMIT_TYPE: hmac
+  BAT_SETTLEMENT_ADDRESS: ${{secrets.BAT_SETTLEMENT_ADDRESS}}
+  DONOR_WALLET_CARD_ID: ${{secrets.DONOR_WALLET_CARD_ID}}
+  DONOR_WALLET_PRIVATE_KEY: ${{secrets.DONOR_WALLET_PRIVATE_KEY}}
+  DONOR_WALLET_PUBLIC_KEY: ${{secrets.DONOR_WALLET_PUBLIC_KEY}}
+  ENCRYPTION_KEY: ${{secrets.ENCRYPTION_KEY}}
+  GEMINI_CLIENT_ID: ${{secrets.GEMINI_CLIENT_ID}}
+  GEMINI_CLIENT_KEY: ${{secrets.GEMINI_CLIENT_KEY}}
+  GEMINI_CLIENT_SECRET: ${{secrets.GEMINI_CLIENT_SECRET}}
+  GEMINI_SERVER: ${{secrets.GEMINI_SERVER}}
+  GEMINI_TEST_DESTINATION_ID: ${{secrets.GEMINI_TEST_DESTINATION_ID}}
+  GRANT_WALLET_CARD_ID: ${{secrets.GRANT_WALLET_CARD_ID}}
+  GRANT_WALLET_PRIVATE_KEY: ${{secrets.GRANT_WALLET_PRIVATE_KEY}}
+  GRANT_WALLET_PUBLIC_KEY: ${{secrets.GRANT_WALLET_PUBLIC_KEY}}
+  UPHOLD_ACCESS_TOKEN: ${{secrets.UPHOLD_ACCESS_TOKEN}}
+  RATIOS_TOKEN: ${{secrets.RATIOS_TOKEN}}
+  UPHOLD_SETTLEMENT_ADDRESS: ${{secrets.UPHOLD_SETTLEMENT_ADDRESS}}
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goversion:
+        - 1.14
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{matrix.goversion}}
+
+    - name: Docker Compose Install
+      uses: KengoTODA/actions-setup-docker-compose@main
```

Comment:
i'm a little concerned that nobody has audited this repo (given that it only has one open issue). is there a more trustworthy alternative?

Reply:
this one [isbang/compose-action](https://github.com/isbang/compose-action) seems to do basically the same thing but came up first in the search results. not many people bring up with compose. i suppose i could do without? would have to tinker with some docker commands though

Reply:
used hashes instead of versions

Reply:
@diracdeltas there is not an official actions/docker-compose from what i can tell though, no